docs/sources/getting-started/glossary.md

```diff
@@ -17,41 +17,72 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
 
 <table>
   <tr>
-    <td style = "vertical-align: top;">Dashboard</td>
-    <td>A set of one or more panels, organized and arranged into one or more rows, that provide an at-a-glance view of related information.</td>
+    <td style="vertical-align: top">Dashboard</td>
+    <td>
+      A set of one or more panels, organized and arranged into one or more rows, that provide an at-a-glance view of related information.
+    </td>
+  </tr>
+  <tr>
+    <td style="vertical-align: top">Data source</td>
+    <td>
+      A file, database, or service providing the data. Grafana supports a several data sources by default, and can be extended to support additional ones through plugins.
```

Comment:
You might add a link to plugins using the last word as the link text.

Reply:
I'll fix this in a separate PR!